### ISSUE TITLE
Grammar editor shows error when the grammar file ends with comment

### ISSUE THREAD

**Issue by SomMeri** [jknack/antlr4ide#116]
Grammar editor shows error if grammar ends with a comment. Error marker text:
- `mismatched character '<EOF>' expecting '\n'`

Following grammar was downloaded from [antlr4 examples](https://github.com/antlr/grammars-v4/tree/master/csv). It shows error on the last line:

```
grammar CSV;

file: hdr row+ ;
hdr : row ;

row : field (',' field)* '\r'? '\n' ;

field
    : TEXT
    | STRING
    |
    ;

TEXT   : ~[,\n\r"]+ ;
STRING : '"' ('""'|~'"')* '"' ; // quote-quote is an escaped quote
```

Antlr itself generates corrects parser, only eclipse plugin gui reports the error. 
Removing last comment removes the error. 
IDE version: 0.3.5

**Comment by HSorensen** [comment]
@arifogel I looked at this. Please review my changes.

**Comment by HSorensen** [comment]
Fixed by pull request #156
